fiasco/ions.py

```diff
@@ -1316,6 +1336,35 @@ def free_free(self, wavelength: u.angstrom) -> u.erg * u.cm**3 / u.s / u.angstro
         return (prefactor * self.charge_state**2 * exp_factor * gf
                 / np.sqrt(self.temperature)[:, np.newaxis])
 
+    @u.quantity_input
+    def free_free_radiative_loss(self) -> u.erg * u.cm**3 / u.s:
+        r"""
+        Free-free continuum radiative losses as a function of temperature.
+
+        The total free-free radiative loss is given by integrating the emissivity over
+        all wavelengths.  The total losses per unit emission measure is then given by
+        (Equation 18 of :cite:`sutherland_accurate_1998`):
+
+        .. math::
+
+            R_{ff}(T_e) = F_{k} \sqrt{(T_{e})} Z^{2} \langle g_{t,ff}\rangle
+
+        where :math: `T_{e}` is the electron temperature, :math: `\F_{k}` is a constant,
+        :math: `Z` is the ionization stage, and :math: `\langle g_{t,ff}\rangle` is the
+        total (wavelength-averaged) free-free Gaunt factor.
+
+        Notes
+        -----
+        The result does not include ionization equilibrium or abundance factors.
+
+        The prefactor :math: `F_{k}` is defined in Equation 19 of :cite:t:`sutherland_accurate_1998`,
+        with a value of 1.42555669e-27 cm$^{5}$ g K$^{-1/2}$ s$^{3}$).
+
+        """
+        prefactor = (16./3**1.5) * np.sqrt(2. * np.pi * const.k_B/(const.hbar**2 * const.m_e**3)) * (const.e.esu**6 / const.c**3)
```

Comment:
Where does the $16/3^{1.5}$ number come from?

Reply:
[Sutherland 1998](https://ui.adsabs.harvard.edu/abs/1998MNRAS.300..321S/abstract) spells out the whole equation.
<img width="437" alt="Screenshot 2024-08-09 at 12 57 01 PM" src="https://github.com/user-attachments/assets/fbf53ef1-6c60-40a0-9a69-7f20dddf1c71">